src/System.Net.Http/src/System/Net/Http/MessageProcessingHandler.cs

```diff
@@ -47,40 +47,46 @@ protected MessageProcessingHandler(HttpMessageHandler innerHandler)
 
                 // We schedule a continuation task once the inner handler completes in order to trigger the response
                 // processing method. ProcessResponse() is only called if the task wasn't canceled before.
-                sendAsyncTask.ContinueWithStandard(task =>
+                var tuple = Tuple.Create(this, tcs, cancellationToken);
```

Comment:
Note: I know the coding guidelines say to not use `var` if the type isn't obvious, but I intentionally left it out here. The full type is

```cs
Tuple<MessageProcessingHandler, TaskCompletionSource<HttpResponseMessage>, CancellationToken>
```

which is pretty long IMO.